src/pages/TasksPage/TasksPage.jsx

```diff
@@ -1,25 +1,42 @@
-import { useEffect, useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import { useNavigate } from 'react-router-dom'
-import { CreateTaskModal } from '../../widgets/CreateTaskModal'
 import Button from '@mui/material/Button'
-import { TaskCard } from '../../entities/TaskCard'
+import { TaskCard } from 'entities/TaskCard'
 import api from '../../shared/service/axios/axiosClient'
 import Typography from '@mui/material/Typography'
+import { EditTaskModal, CreateTaskModal } from 'widgets/TaskModal'
 
 import { Loading } from '../../shared/components/Loading'
 import { noData } from './tasksPage.model'
 import { Header } from '../../widgets/Header'
 import { useAuth } from '../../shared/hooks/useAuth'
+import { useModalState } from '../../shared/hooks/useModalState'
+import { CSVModal } from 'widgets/CSVModal'
 
 export const TasksPage = () => {
-  const [open, setOpen] = useState(false)
   const [tasks, setTasks] = useState([])
   const navigate = useNavigate()
   const [loading, setLoading] = useState(false)
   const { isLoading, isAuth, user } = useAuth()
 
-  const handleOpen = () => {
-    setOpen(true)
+  const [isCreateOpen, openCreateModal, closeCreateModal] = useModalState()
+  const [isEditOpen, openEditModal, closeEditModal] = useModalState()
+  const [isCSVOpen, openCSVModal, closeCSVModal] = useModalState()
+
+  const [getId, setGetId] = useState('')
+
+  const handleCreateOpen = () => {
+    openCreateModal()
+  }
```

Comment:
const handleCreateOpen = () => openCreateModal()

Reply:
Returning a void expression from an arrow function shorthand is forbidden. Please add braces to the arrow function.eslint[@typescript-eslint/no-confusing-void-expression](https://typescript-eslint.io/rules/no-confusing-void-expression)